package.json

```diff
@@ -2,7 +2,9 @@
   "name": "@ampath-feature-flag-mgmt/source",
   "version": "0.0.0",
   "license": "MIT",
-  "scripts": {},
+  "scripts": {
+    "start": "npx nx serve back-end"
```

Comment:
you can change this to something like "start:be" to distinguish with starting front-end app